assets/cubyz/blocks/dynamite/block.zig.zon

```diff
@@ -0,0 +1,12 @@
+.{
+	.tags = .{.canBeIgnited, .explosionMedium},
```

Comment:
Shouldn't the explosion radius be specifiable as a separate property or something? I'm sure we can spare a `u8` per block type for that.

Reply:
That is wasteful and not a sustainable model. This property will be used by 2 blocks in base game (maybe 5 if you let us be creative). And it is completely irrelevant to performance. Will you add a field for every property that makes sense for 3 blocks? And what if a modification wants to add one of those? Instead we could have a sparse set of hash maps string to zon and just solve that problem in generic way.

Reply:
Well, I think it's only a matter of time until addon creators want to make nuclear explosions with this.
Either way this can be done in the future if you want.